pkg/controller/appdefinition/sleep_image.go

```diff
@@ -0,0 +1,8 @@
+//go:build image
+
+package appdefinition
+
+import _ "embed"
+
+//go:embed embed/acorn-sleep
```

Comment:
Is it possible for an app's underlying pods to get scheduled to a node with a different arch than the one the runtime controller is running on? I'm not sure if we even support multi-arch setups, but if we do, this could be an issue.

Reply:
Yeah, we currently don't support multiple architectures. We'll have to cross that bridge when we come to it.